packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -608,6 +608,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       this.handleGotoValueSelectedFilterChanged.bind(this);
     this.handleGotoValueChanged = this.handleGotoValueChanged.bind(this);
     this.handleGotoValueSubmitted = this.handleGotoValueSubmitted.bind(this);
+    this.onViewportUpdate = this.onViewportUpdate.bind(this);
```

Comment:
Name this `handleViewportUpdate` instead. That's the naming convention that React tends to use in their docs: https://react.dev/learn/responding-to-events
`on*` for the prop name, and then `handle*` for the handler's name.